tools/test-npm.sh

```diff
@@ -17,6 +17,19 @@ rm -rf test-npm
 # make a copy of deps/npm to run the tests on
 cp -r deps/npm test-npm
 
+
+## a hopefully temporary hack to use the local $NODE binary when calling node(1)
+# make a link to $NODE as test-npm/node
+ln -f $NODE test-npm/node
```

Comment:
This may be an issue cross-platform.

My OS X `ls` cannot do relative symlinks (`-r`), but linux does not appear to like hard links.